salt/config/schemas/esxvm.py

```diff
@@ -290,8 +304,10 @@ class ESXVirtualMachineInterfaceSchemaItem(ComplexSchemaItem):
     )
     adapter = StringItem(
         title="Virtual Machine Network Adapter",
-        description="Unique name of the network adapter, "
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique name of the network adapter, "
+            "for creation an arbitrary value should be specified"
```

Comment:
Same thing here.

Reply:
This one looks good, no?

Reply:
Yup!  Missed the space at the end on the first line.